src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/summaries/temperature/ClusterDimensionalSummary.java

```diff
@@ -70,13 +71,19 @@
     /**
      * This is the mean temperature for this dimension over all the nodes in the cluster.
      */
-    private TemperatureVector.NormalizedValue meanTemperature;
+    private TemperatureVector.VectorValues meanTemperature;
+
+    /**
+     * This is the average value used over shards.
+     */
+
+    private double avgMetricValueUsedOverNodes;
 
     /**
      * meanTemperature is a normalized value. The total tells us if this is something that one
      * should be concerned about.
      */
-    private double totalUsage;
+    private double totalMetricValueUsedOverCluster;
```

Comment:
Does the substring `OverCluster` and `OverNodes` signify something different ? If not, then we can remove them as the name of the class tells us that this is something over all the nodes? Thoughts ?